Dockerfile

```diff
@@ -90,3 +91,40 @@ WORKDIR /home/${SERVICE_USER}
 RUN forest -V && forest-cli -V && forest-tool -V
 
 ENTRYPOINT ["forest"]
+
+# A fat image contains forest binaries and fil proof parameter files under $FIL_PROOFS_PARAMETER_CACHE
+FROM ubuntu:22.04 as fat-image
```

Comment:
Would `FROM slim-image` so that less code is duplicated?

Reply:
Good idea, fixed